src/index.js

```diff
@@ -1223,6 +1234,35 @@ export default class AuthnWidget {
     this.pollCheckGet(this.store.getStore().verificationCode, 5000);
   }
 
+  postContinue() {
+    this.store.dispatch('POST_FLOW', 'continue', '{}')
+      .catch(() => this.generalErrorRenderer(AuthnWidget.COMMUNICATION_ERROR_MSG));
+  }
+
+  pollLinkStatus() {
+    this.pollLink(5000);
+  }
+
+  async pollLink(timeout) {
+    let fetchUtil = this.store.fetchUtil;
```

Comment:
I'll need to change this method in the near future :) can you introduce a new method in the store to do the bulk of this operation? aka perform a poll and return the state.

Reply:
Added a new poll() method to the datastore for this functionality. Let me know if that's not what we are looking for.

Reply:
Yea, thanks so much.